doc/api/next_api_changes/2019-07-18-SL.rst

```diff
@@ -0,0 +1,4 @@
+Reduced default value of `axes.formatter.limits`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changed the default value of `axes.formatter.limits` from -7, 7 to -5, 6 for better readability.
```

Comment:
@sameshl If you get a chance could you add a `.. plot` example to the API change note?  It greatly helps people who are skimming the release note to understand the change!